src/pages/Index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+
+function Index() {
```

Comment:
If this is our landing or "home" page you can name it "Landing" or "Landing Page", leave index name for index.html